process/mock/accountWrapperFake.go

```diff
@@ -21,6 +23,51 @@ type AccountWrapMock struct {
 	SetCodeWithJournalCalled     func(codeHash []byte) error `json:"-"`
 }
 
+// AddToBalance -
+func (awm *AccountWrapMock) AddToBalance(_ *big.Int) error {
```

Comment:
might change the filename: accountWrapperMock.

Reply:
Done.